packages/resource-deployment/templates/batch-account.template.json

```diff
@@ -118,10 +118,10 @@
                         "imageReference": {
                             "publisher": "microsoft-azure-batch",
                             "offer": "ubuntu-server-container",
-                            "sku": "16-04-lts",
+                            "sku": "18-04-lts",
```

Comment:
@dbjorge  have you validated this change by deploying it using https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_release?_a=releases&view=mine&definitionId=12

Also, since you are changing the image, we need to drop nodes & create again. (this can be achieved by setting the release environment variable dropPools=true (Else the deployment will fail)